.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,137 @@
+on:
+  # Branch deletion
+  delete:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+env:
+  PRODUCTION_BRANCH: "master"
+  REPO_NAME: ${{ github.event.repository.name }}
+  PULL_REQUEST_NUMBER: ${{ github.event.number }}
+  GITHUB_ACTION: ${{ github.event.action }}
+  AWS_REGION: "us-east-1"
+  SERVICE_NAME: "badges"
+  REPO_OWNER: "FormidableLabs"
+  GITHUB_HASH: ${{ github.event.after }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  base:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the Repository
+        uses: actions/checkout@v2
+
+      - name: Set environment variables
+        # https://stackoverflow.com/questions/59882715/use-environment-variable-in-github-action-if
+        # https://www.techiediaries.com/get-current-branch-name-github-actions-github_ref-bash/
+        run: |
+          if [[ $GITHUB_EVENT_NAME != 'pull_request' ]]; then
+            BRANCH_NAME=$(echo "${GITHUB_REF#refs/heads/}" | sed 's/\//-/')
+            echo "BRANCH_NAME=$BRANCH_NAME" >> $GITHUB_ENV
+            if [[ "${GITHUB_REF#refs/heads/}" == $PRODUCTION_BRANCH ]]; then
+              TIER="prod"
+              STAGE="production"
+            else
+              TIER="nonprod"
+              STAGE="development"
+            fi
+          else
+            TIER="nonprod"
+            STAGE="development"
+          fi
+          echo "TIER=$TIER" >> $GITHUB_ENV
+          echo "STAGE=$STAGE" >> $GITHUB_ENV
+
+      - name: Run the install script
+        run: bash scripts/install.sh
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Run the yarn ci script
+        run: yarn ci
+
+      - uses: actions/upload-artifact@master
+        with:
+          name: serverless-directory
+          path: .serverless
+          retention-days: 1
+
+      - name: Post comment on pull request
+        uses: actions/github-script@0.9.0
+        if: github.event_name == 'pull_request'
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const endpoint=`https://${process.env.REPO_NAME}-${process.env.TIER}-pr${process.env.PULL_REQUEST_NUMBER}.freetls.fastly.net`
+            examples = [
+              'size/github/FormidableLabs/react-fast-compare/master/index.js',
+              'size/npm/victory/dist/victory.min.js?gzip=true',
+              'browsers?firefox=20,26&iexplore=!8,-9,10',
+              'travis/infernojs/inferno/sauce/Havunen?name=InfernoJS',
```

Comment:
Can we ticket for the future that this one needs refactoring / removal as I don't think Inferno uses Sauce anymore and the badge is just "unknown"?

Reply:
Sure